### PR TITLE
[CSE-86] Use yarn to speed up build

### PR DESCRIPTION
**Pull request by sectore**
Incl. update of `README` for using `yarn` and to have a better overview about production build.

@domenkozar Please note, that `travis` might be failed, because we are using `yarn` now.

### CODE REVIEW

File: frontend/scripts/build.sh [hunk 1]
```diff
@@ -18,8 +18,8 @@ else
 fi
 popd
 nix-shell --run "rm -rf .psci_modules/ .pulp-cache/ node_modules/ bower_components/ output/"
-nix-shell --run "npm install"
+nix-shell --run "yarn install"
```
Comment (domenkozar) [inline frontend/scripts/build.sh]: You'll have to put `yarn` also on the second line

Comment (sectore) [inline frontend/scripts/build.sh]: @domenkozar To have something like this `#! nix-shell -j 4 -i bash -p stack git yarn` ?

Comment (domenkozar) [inline frontend/scripts/build.sh]: Correct

Comment (domenkozar) [inline frontend/scripts/build.sh]: Or even better, put it in `frontend/default.nix`

Comment (sectore) [inline frontend/scripts/build.sh]: @domenkozar All line into `frontend/default.nix`?

Comment (domenkozar) [inline frontend/scripts/build.sh]: `buildInputs = with hspkgs; [ nodejs yarn nodePackages.bower purescript purescript-derive-lenses ];`

Comment (sectore) [inline frontend/scripts/build.sh]: @domenkozar ⬇️

**Pull request merged**